### PR TITLE
Update autoflake parameters to remove unused imports

### LINKED ISSUE #296: [development] make lint-fix doesn't fix linting issue
### Summary
make lint-fix doesn't fix linting issue

### Steps to Reproduce it

I have redundant imports in `dsketch.py` file and `make lint-fix` did not address the issue

### Example

Here's the permanent link to the extra import: https://github.com/whylabs/whylogs/blob/942093f4de4af9846d728efffb0923d199e73a5d/src/whylogs/util/dsketch.py#L15

Failed un:
https://github.com/whylabs/whylogs/pull/295/checks?check_run_id=3444995360

## What is the current *bug* behavior?

Same as above

### Relevant logs and/or screenshots

`dsketch` is listed in the log of linting. `make format` doesn't do anything either (no meaningful logs)
```
poetry run autoflake --in-place --remove-unused-variables ./src/whylogs/core/statistics/schematracker.py ./src/whylogs/core/statistics/hllsketch.py ./src/whylogs/core/statistics/counterstracker.py ./src/whylogs/core/statistics/datatypes/variancetracker.py ./src/whylogs/core/statistics/datatypes/floattracker.py ./src/whylogs/core/statistics/datatypes/__init__.py ./src/whylogs/core/statistics/datatypes/integertracker.py ./src/whylogs/core/statistics/__init__.py ./src/whylogs/core/statistics/thetasketch.py ./src/whylogs/core/statistics/constraints.py ./src/whylogs/core/statistics/stringtracker.py ./src/whylogs/core/statistics/numbertracker.py ./src/whylogs/core/metrics/__init__.py ./src/whylogs/core/metrics/regression_metrics.py ./src/whylogs/core/metrics/nlp_metrics.py ./src/whylogs/core/metrics/model_metrics.py ./src/whylogs/core/metrics/confusion_matrix.py ./src/whylogs/core/types/__init__.py ./src/whylogs/core/types/typeddataconverter.py ./src/whylogs/core/summaryconverters.py ./src/whylogs/core/__init__.py ./src/whylogs/core/annotation_profiling.py ./src/whylogs/core/image_profiling.py ./src/whylogs/core/datasetprofile.py ./src/whylogs/core/flatten_datasetprofile.py ./src/whylogs/core/model_profile.py ./src/whylogs/core/columnprofile.py ./src/whylogs/app/config.py ./src/whylogs/app/output_formats.py ./src/whylogs/app/metadata_writer.py ./src/whylogs/app/session.py ./src/whylogs/app/__init__.py ./src/whylogs/app/writers.py ./src/whylogs/app/logger.py ./src/whylogs/app/utils.py ./src/whylogs/proto/messages_pb2.py ./src/whylogs/proto/__init__.py ./src/whylogs/proto/summaries_pb2.py ./src/whylogs/proto/constraints_pb2.py ./src/whylogs/util/time.py ./src/whylogs/util/__init__.py ./src/whylogs/util/stats.py ./src/whylogs/util/dsketch.py ./src/whylogs/util/util_functions.py ./src/whylogs/util/protobuf.py ./src/whylogs/util/varint.py ./src/whylogs/util/data.py ./src/whylogs/_version.py ./src/whylogs/io/__init__.py ./src/whylogs/io/file_loader.py ./src/whylogs/io/local_dataset.py ./src/whylogs/features/transforms.py ./src/whylogs/features/__init__.py ./src/whylogs/features/autosegmentation.py ./src/whylogs/mlflow/__init__.py ./src/whylogs/mlflow/model_wrapper.py ./src/whylogs/mlflow/patcher.py ./src/whylogs/mlflow/sklearn.py ./src/whylogs/__init__.py ./src/whylogs/cli/cli_text.py ./src/whylogs/cli/__init__.py ./src/whylogs/cli/cli.py ./src/whylogs/cli/utils.py ./src/whylogs/cli/init.py ./src/whylogs/cli/demo_cli.py ./src/whylogs/viz/browser_viz.py ./src/whylogs/viz/__init__.py ./src/whylogs/viz/matplotlib/__init__.py ./src/whylogs/viz/matplotlib/visualizer.py ./src/whylogs/viz/base.py ./src/whylogs/viz/visualizer.py ./src/whylogs/logs/__init__.py ./src/whylogs/whylabs_client/wrapper.py ./src/whylogs/whylabs_client/__init__.py ./tests/unit/conftest.py ./tests/unit/test_nothing2.py ./tests/unit/core/statistics/test_schematracker.py ./tests/unit/core/statistics/datatypes/test_inttracker.py ./tests/unit/core/statistics/datatypes/__init__.py ./tests/unit/core/statistics/datatypes/test_floattracker.py ./tests/unit/core/statistics/datatypes/test_variancetracker.py ./tests/unit/core/statistics/__init__.py ./tests/unit/core/statistics/test_counterstracker.py ./tests/unit/core/statistics/test_constraints.py ./tests/unit/core/statistics/test_hllsketch.py ./tests/unit/core/statistics/test_numbertracker.py ./tests/unit/core/statistics/test_thetasketch.py ./tests/unit/core/statistics/test_stringtracker.py ./tests/unit/core/metrics/test_confusion_matrix.py ./tests/unit/core/metrics/__init__.py ./tests/unit/core/metrics/test_regression_metrics.py ./tests/unit/core/metrics/test_model_metrics.py ./tests/unit/core/metrics/test_nlp_metrics.py ./tests/unit/core/types/__init__.py ./tests/unit/core/types/test_typeddataconverter.py ./tests/unit/core/test_datasetprofile_datatypes.py ./tests/unit/core/test_image_profiling.py ./tests/unit/core/test_datasetprofile.py ./tests/unit/core/test_rect.py ./tests/unit/core/__init__.py ./tests/unit/core/test_datasetprofile_metrics.py ./tests/unit/core/test_columnprofile.py ./tests/unit/core/test_annotation_profiling.py ./tests/unit/core/test_summaryconverters.py ./tests/unit/core/test_model_profile.py ./tests/unit/app/test_segments.py ./tests/unit/app/test_logger_image.py ./tests/unit/app/test_session.py ./tests/unit/app/test_writers.py ./tests/unit/app/test_log_rotation.py ./tests/unit/app/test_logger_metrics.py ./tests/unit/util/test_stats.py ./tests/unit/util/test_varint.py ./tests/unit/util/__init__.py ./tests/unit/util/test_protobuf.py ./tests/unit/util/test_dsketch.py ./tests/unit/util/test_dsketch_frequentnumbers.py ./tests/unit/util/test_dsketch_frequentitems.py ./tests/unit/io/test_file_loader.py ./tests/unit/io/__init__.py ./tests/unit/io/test_local_dataset.py ./tests/unit/features/test_autosegmentation.py ./tests/unit/features/__inti__.py ./tests/unit/features/test_image_transforms.py ./tests/unit/mlflow/__init__.py ./tests/unit/mlflow/test_patcher.py ./tests/unit/mlflow/test_mlflow.py ./tests/unit/__init__.py ./tests/unit/cli/__init__.py ./tests/unit/cli/test_cli.py ./tests/unit/viz/test_profile_viewer.py ./tests/unit/viz/__init__.py ./tests/unit/viz/test_viz.py ./tests/unit/logs/test_diplay.py ./tests/unit/logs/__init__.py ./tests/conftest.py ./tests/__init__.py ./tests/component/conftest.py ./tests/component/app/__init__.py ./tests/component/app/bucket/__init__.py ./tests/component/app/bucket/whylogs/__init__.py ./tests/component/app/test_logger.py ./tests/component/__init__.py ./tests/component/test_nothing.py ./tests/helpers/__init__.py ./tests/helpers/testutil.py ./test_notebooks/conftest.py ./test_notebooks/__init__.py ./test_notebooks/notebook_tests.py
```

### PR DESCRIPTION
**Pull request by jamie256**
## Description

Small change to autoflake parameters so that when we run:
Make lint-fix

that will remove unused imports. Also updated files accordingly. This resolves #296

### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [x] Conform by the style guides, by using formatter
* [ ] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

**Review by andyndang — APPROVED** [review]
LGTM!
